gluten-core/src/main/scala/io/glutenproject/backendsapi/TransformerApi.scala

```diff
@@ -63,4 +64,7 @@ trait TransformerApi {
   def getSupportExpressionClassName: util.Set[String] = {
     util.Collections.emptySet()
   }
+  def getAggregateOutput(plan: SparkPlan) : Seq[Attribute] = {
```

Comment:
Can we use `HashAggregateExec` as input for this function? With `SparkPlan` used, this function looks general and not restricted to the function name `getAggregateOutput`.

Reply:
done

Reply:
rename the method name.This function is to get the plan node output